tests/device_tests/test_session_id_and_passphrase.py

```diff
@@ -97,7 +97,7 @@ def test_session_with_passphrase(client):
     assert _get_xpub(client, passphrase="A") == XPUB_PASSPHRASES["A"]
 
 
-@pytest.mark.setup_client(passphrase=True)
+@pytest.mark.setup_client(passphrase="")
```

Comment:
please keep `passphrase=True` in this file

Reply:
Fixed in 40530e8